set_ar5search_env.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
We might want to use an `.env.production` file for this.
Might also want to replace the defaults (that are currently set somewhere in `next.config.js` IIRC) with a plain `.env` file.

See https://nextjs.org/docs/basic-features/environment-variables#environment-variable-load-order.

Reply:
Not sure this has to be done now, feel free to leave as is for now and make an issue out of this.

Reply:
Just noticed, we might also have to prefix some of the variables with [`NEXT_PUBLIC_`](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser) to get them to show up in the client.

Reply:
> We might want to use an .env.production file for this.

We will can have different env variables for each instance. Eg, nLab, arXiv, etc. so having a single file 


> Just noticed, we might also have to prefix some of the variables with [NEXT_PUBLIC_](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser) to get them to show up in the client.

We are using the older [runtime config](https://nextjs.org/docs/api-reference/next.config.js/environment-variables). The new scheme you referenced is more convenient.

> Not sure this has to be done now, feel free to leave as is for now and make an issue out of this.

There is probably a better solution for managing variables. Created Issue #36

Reply:
Sounds good.